detections/endpoint/powershell_powershell_script_block_with_url_chain.yml

```diff
@@ -0,0 +1,74 @@
+name: PowerShell Script Block With URL Chain
+id: 4a3f2a7d-6402-4e64-a76a-869588ec3b57
+version: 1
+date: '2023-06-13'
+author: Steven Dick
+status: production
+type: TTP
+description: The following analytic identifies a suspicious PowerShell script execution via EventCode 4104 that contains multiple URLs within a function or array.
+ This is typically found in obfuscated PowerShell or PowerShell executing embedded .NET or binary files that are attempting to download 2nd stage payloads. 
+ During triage, review parallel processes within the same timeframe. Review the full script block to identify other related artifacts.

```

Comment:
updated some yaml formatting since the line breaks were causing some validation failures.